lua/languages/global/languages/css.lua

```diff
@@ -37,19 +28,8 @@ language_configs["lsp"] = function()
             handlers = languages_setup.show_line_diagnostics()
         }
     end
-    local ok, cssls = lsp_installer_servers.get_server("cssls")
-    if ok then
-        if not cssls:is_installed() then
-            cssls:install()
-            lsp_installer.on_server_ready(
-                function()
-                    start_cssls()
-                end
-            )
```

Comment:
This will actually set up a handler that is executed every time _any server_ is ready. As it's implemented now, this assumes that `cssls` will be the only server this handler is called for. This is compounding since it seems like every server module in lvim sets up a new handler, meaning these handlers will be called `n * m` times